pylint/test/input/func_w1125.py

```diff
@@ -0,0 +1,9 @@
+"""Unittests for W1125 (kw args before *args)"""
```

Comment:
We stopped adding tests in this directory. All new tests should go in ``functional`` directory. Please check there for examples on how a test should look like.